native/src/main/java/io/ballerina/persist/objects/Entity.java

```diff
@@ -43,6 +46,17 @@ public Entity(String[] keys, String tableName, Optional<String> module) {
         this.module = module;
     }
 
+    public String getNamePrefix(boolean withTail) {
+        if (!module.isEmpty() && withTail) {
+            return module.get() + "Entities";
+        } else if (!module.isEmpty() && !withTail) {
+            return module.get().substring(0, 1).toUpperCase() + module.get().substring(1);
+        } else if (module.isEmpty() && !withTail) {
+            return "";
+        }
+        return "entities";
```

Comment:
```suggestion
        return modulePrefix;
```

Dont use string when you can use constants/module level variables

Reply:
updated in [eac1fc8](https://github.com/ballerina-platform/persist-tools/pull/23/commits/eac1fc89ac6d1d3cd9a9387fce669b33074b9ced)